src/averageAges.js

```diff
@@ -37,7 +47,25 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+
+  if (withChildren === true) {
```

Comment:
```suggestion
  if (withChildren === true) {
```
Тут також давай замінимо if на тернарник